README.md

```diff
@@ -1,6 +1,6 @@
 # Crafter CMS
 
-Crafter CMS is an open source content management system for Web sites, mobile apps, VR and more. You can learn more about Crafter here: http://docs.craftercms.org/en/3.1/index.html
+Crafter CMS is an open source content management platform for building digital experience applications (React, Vue, Angular, HTML, AR/VR, Mobile, Headless, …).  You can learn more about Crafter here: http://docs.craftercms.org/en/3.1/index.html
```

Comment:
Why not use the same initial blurb we now have in 3.1 docs? (not the whole thing, just the first section).